packages/kit/src/core/sync/create_manifest_data/index.spec.js

```diff
@@ -173,8 +173,16 @@ test('creates routes with layout', () => {
 
 test('succeeds when routes does not exist', () => {
 	const { nodes, routes } = create('samples/basic/routes');
-	assert.equal(nodes, []);
-	assert.equal(routes, []);
+	assert.equal(nodes.map(simplify_node), [
+		{ component: 'layout.svelte' },
+		{ component: 'error.svelte' }
+	]);
+	assert.equal(routes.map(simplify_route), [
+		{
+			id: '',
+			pattern: '/^$/'
+		}
+	]);
```

Comment:
The test shows that nodes for the root layout and error are created. This means that the running code later on can safely assume these to be always present. Before that, the manifest was generated without these, causing `dev` and `build` to fail with "cannot read properties of nodes[0].component" or sth like that. Adding the default nodes even in case of a missing routes folder seems like the better option, even if it seems somewhat strange at first. I'll add a comment to the manifest about this.